pkg/debezium/typeutil/helpers.go

```diff
@@ -17,29 +17,6 @@ import (
 	"github.com/doublecloud/transfer/pkg/util"
 )
 
-//---------------------------------------------------------------------------------------------------------------------
-// it's fixed in higher version of debezium (in 1.1 this bug is present, in 1.8 absent)
-// so, actual function - changeItemsBitsToDebeziumHonest
-
-func changeItemsBitsToDebeziumWA(bits string) string {
```

Comment:
better to delete this method separately, not sure that it's completely useless (at least as documentation).